src/main/java/io/github/christophermanahan/carnitas/Serializer.java

```diff
@@ -15,15 +15,15 @@
         ).getBytes();
     }
 
-    private String statusLine(HTTPResponse response) {
-        return HTTPResponse.VERSION + " " + response.status().code + Serializer.CRLF;
+    private String statusLine(Response response) {
+        return Response.VERSION + " " + response.status().code + Serializer.CRLF;
     }
 
-    private String format(List<String> headers) {
+    private String format(TreeSet headers) {
```

Comment:
Does this _need_ to be a `TreeSet` or could it be more generic? (i.e. `Set<String>`).